charts/sonarqube/templates/_pod.tpl

```diff
@@ -411,6 +448,14 @@ spec:
         items:
           - key: install_plugins.sh
             path: install_plugins.sh
+    {{- end }}
+        {{- if .Values.jdbcOverwrite.oracleJdbcDriverURL }}
```

Comment:
few extra space on the left side of the if ?

Reply:
Addressed